lib/tool_shed/webapp/config.py

```diff
@@ -34,6 +38,18 @@ class ConfigurationError(Exception):
 class ToolShedAppConfiguration(BaseAppConfiguration):
     default_config_file_name = 'tool_shed.yml'
 
+    def _load_schema(self):
+        return AppSchema(TOOLSHED_CONFIG_SCHEMA_PATH, TOOLSHED_APP_NAME)
+
+    def _update_raw_config_from_kwargs(self, kwargs):
+        pass  # Intentionally does nothing. To enable, remove + call base __init__().
+
+    def _create_attributes_from_raw_config(self):
+        pass  # Intentionally does nothing. To enable, remove + call base __init__().
+
+    def _resolve_paths(self, paths_to_resolve):
+        pass  # Intentionally does nothing. To enable, remove + call base __init__().
+
```

Comment:
Why doesn't the TS use these?

Reply:
Oh, but it's about to :-) https://github.com/galaxyproject/galaxy/pull/9379